.github/workflows/ci.yml

```diff
@@ -116,3 +116,21 @@ jobs:
         with:
           command: check
           args: --manifest-path ravedude/Cargo.toml
+
+micronucleus_runner:
+    name: "micronucleus-runner"
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Install Rust
+        uses: actions-rs/toolchain@v1
```

Comment:
`action-rs/toolchain` is unmaintained.  `actions-rust-lang/setup-rust-toolchain@v1` is one alternative